recipes/nsc/recipe.yaml

```diff
@@ -0,0 +1,56 @@
+context:
+  version: "0.0.437"
+
+package:
+  name: nsc
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/namespacelabs/foundation/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: a229b1c973539323347f04e0028af5fade498e8ec4821b853225896cbd490338
+
+build:
+  number: 0
+  script:
+    - go build -ldflags="-s -w -X namespacelabs.dev/foundation/internal/cli/version.Tag=v${{ version }}" -o ${{ PREFIX }}/bin/nsc${{ ".exe" if win }} ./cmd/nsc
+
+    # Generate shell completions (only on Unix and when target platform equals build platform)
+    - if: unix and target_platform == build_platform
```

Comment:
shouldn't this also work with emulation when compiling to linux-aarch64, linux-ppc64le?